charts/kueue/tests/manager_config_test.yaml

```diff
@@ -0,0 +1,105 @@
+suite: test manager-config configmap
```

Comment:
@cmtly @mbobrovskyi Are these tests triggered automatically on CI, or just manually? 

If the tests are just running manually, then we should in a follow up extend the verify goal to run them on CI.

Reply:
It is running by verify:

https://github.com/kubernetes-sigs/kueue/blob/ae0cdb297e9386d57e3f2722078775a5e17b1f9b/Makefile#L248-L251

https://github.com/kubernetes-sigs/kueue/blob/ae0cdb297e9386d57e3f2722078775a5e17b1f9b/Makefile#L222-L225

Reply:
Awesome, thanks for clarifying!